<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserItem</class>
 <widget class="QWidget" name="UserItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>241</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="headImgPushBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="signedNameLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="friendAccLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>0</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
